### DISCUSSION TITLE
Looking for Data

### DISCUSSION THREAD

**Discussion by LucasSnyTU** [pysal/spaghetti#714] [Q&A] [answered]
Hi all!
I'm learning how to use pysal's Spaghetti package and have been working with your tutorial at https://pysal.org/spaghetti/notebooks/quickstart.html.
I was hoping someone could send me the .shp files used in the example (streets, schools, and crime). Files I have myself are large and take a long time to process. I would love smaller files to learn with.

**Comment by jGaboardi** [comment, marked as answer]
@LucasSnyTU thanks for your interest in `spaghetti`! All data used in the tutorials can be obtained via `libpysal.examples`. For example, cell 4 of the [Network point pattern attributes](https://pysal.org/spaghetti/notebooks/pointpattern-attributes.html) tutorial calls and reads:

```python
libpysal.examples.get_path("streets.shp")
```

See [here](https://pysal.org/libpysal/notebooks/examples.html) for more information specifically on the `libpysal.examples` submodule.
